modules/serverpod_auth/serverpod_auth_server/lib/src/business/email_auth.dart

```diff
@@ -216,6 +216,10 @@ class Emails {
       where: (t) => t.userId.equals(userId),
     );
     if (auth == null) {
+      session.log(
+        'userId: \'$userId\' is invalid!',
```

Comment:
Suggestion: Change the outer ' to ". Then you don't have to escape them inside the string.

Reply:
Done!